x/crosschain/keeper/grpc_query_cctx_rate_limit_test.go

```diff
@@ -470,6 +476,7 @@ func TestKeeper_RateLimiterInput_Errors(t *testing.T) {
 func TestKeeper_ListPendingCctxWithinRateLimit(t *testing.T) {
 	// create sample TSS
 	tss := sample.Tss()
+	zetaChainID := chains.ZetaTestnetChain.ChainId
```

Comment:
We should make some tests using other network types if possible

Reply:
I'm going to create one more test

Reply:
https://github.com/zeta-chain/node/pull/2256/commits/0c9a7cb8774418fb1e5b4879f69ce010337995c0